golang/internal/helper/image/image_unit_test.go

```diff
@@ -13,35 +13,50 @@ import (
 	"github.com/dyrector-io/dyrectorio/protobuf/go/agent"
 )
 
-func TestRegistryUrl(t *testing.T) {
-	auth := &imageHelper.RegistryAuth{
-		URL: "test",
-	}
-
-	url := imageHelper.GetRegistryURL(nil, auth)
-	assert.Equal(t, url, "test")
+type RegistryTestCase struct {
```

Comment:
The original implementation had info about what is being tested stored in the name of the function, this way now that information is lost, please add a name or description field to this struct and use it as a t.Run, if the test ever goes down, it should help identifying the problem if this test ever starts to fail somehow.

Reply:
Added a name field

Reply:
Neat! :+1: